apps/meeting/src/containers/MeetingForm/index.tsx

```diff
@@ -250,6 +253,14 @@ const MeetingForm: React.FC = () => {
         checked={keepLastFrameWhenPaused}
         onChange={toggleKeepLastFrameWhenPaused}
       />
+      <FormField
+        field={Checkbox}
+        label="Skip default meeting join device listing and selection"
```

Comment:
The application is not acting as it says here when selecting this option. Thought `MeetingManager.join()` does not list and select the devices, in the `DeviceSetup` page since it uses `MicSelection` and `CameraSelection` there's still device permission prompts and list the devices.

Reply:
I agree, but technically we want to somehow map it to skipping `meetingManager.join` API device listing and selection. Thus, I worded it so.

Reply:
In our meeting join flow, we have below steps that lists devices:
1. `meetingManager.join`
2. `DeviceSelections` in Device Setup page
3. `MeetingControls` in Meeting page

With or without using `skipDeviceSelection`, this demo lists devices anyway, but it does skip the device selection. We can't skip listing device from meeting demo anyway, so I think changing the label to "Skip device selection" (not mentioning listing device) makes more sense. Though from API perspective we do skip device listing in `mm.join`.

This flow is bit confusing because from application perspective, we skip device selection (select first device) and automatically select first device again. Considering we have builders refer to our demo for implementation, I think it's better to make this use case clear.

Ideally we should get rid of `DeviceSelection` and `MeetingControls` so we can actually stop device listing and selection from application level.

Reply:
So does "Skip default meeting join device selection" sounds appropriate for our use case?

Reply:
I slightly prefer "Skip device selection". Imaging you are a user that does't know the details of SDK. "Skip default meeting join device selection" could be confusing:
1. what is default meeting join device selection? e.g., select most recent device, select first device, ...
2. If we skip it, will this demo select any device or not?

In this flow the result is that no device is selected. So "Skip device selection"or "join w/o selecting devices" sounds more appropriate.

I'm fine as long as we don't mention "list device", decision is up to you : ).

Reply:
I have removed the `DefaultDeviceSelection` and updated the `infoText` to select the devices manually. I believe either way its going to be implicit and builder will need to be made aware on the `skipDevcieSelection` flag which is only related to `MeetingManager.Join` API. I think it is important to call "meeting join device selection" as that is the flag's exact purpose.